packages/node-build-scripts/add-js-version.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# This script adds the Version of the corresponding javascript package in the line 6 of the generated css.
```

Comment:
I think we should add a disclaimer like: "DISCLAIMER: This only works in Mac OSx"

Reply:
After my latest commit now you will get this error:
![captura de pantalla 2018-10-17 a las 16 16 52](https://user-images.githubusercontent.com/24293766/47092925-57e5c180-d228-11e8-9323-8dd0678e781c.png)
Is it ok?

Reply:
Perfect :)